compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -382,24 +378,24 @@ execIde telemetry (Debug debug) enableScenarioService ghcOpts mbProfileDir (from
                       Logger.GCP.logOptOut gcpState
                       f loggerH
                   Undecided -> f loggerH
-          opts <- defaultOptionsIO (Just lfVersion)
-          initPackageDb opts (InitPkgDb True)
-          dlintDataDir <-locateRunfiles $ mainWorkspace </> "compiler/damlc/daml-ide-core"
-          opts <- pure $ opts
+          initPackageDb options (InitPkgDb True)
```

Comment:
You are missing a call to `mkOptions` before calling this which is probably the reason why none of the modules in the standard library are found (yes this API is horrible).